prosody/Dockerfile

```diff
@@ -41,21 +40,15 @@ RUN set -x && \
       sasl2-bin \
       libsasl2-modules-ldap \
       lua-cyrussasl \
-      lua-inspect \
       lua-ldap \
-      lua-luaossl \
-      lua-sec \
       lua-unbound && \
-    apt-dpkg-wrap apt-get -d install -y jitsi-meet-prosody && \
-    dpkg -x /var/cache/apt/archives/jitsi-meet-prosody*.deb /tmp/pkg && \
-    rm /tmp/pkg/usr/share/jitsi-meet/prosody-plugins/mod_smacks.lua && \
```

Comment:
We still need the remove, to use the prosody provided module.

Reply:
Re-added the removal.